### ISSUE TITLE
Unable to install plugin in Linux Ubuntu 16.04 LTS

### ISSUE THREAD

**Issue by scalahub** [dcaoyuan/nbscala#172]
I get this error 
Some plugins require plugin External Execution Base API to be installed.
The plugin External Execution Base API is requested in implementation version 201510222201.  The following plugins are affected:       Scala Console       Scala SBT Project

What is wrong? I have just got a brand new linux laptop. So its definitely something wrong in the plugin.

**Comment by akochnev** [comment]
@scalahub  - can you provide further details of the version of NetBeans and where you're installing the plugin from ?

**Comment by scalahub** [comment]
Product Version: NetBeans IDE 8.1 (Build 20151231-debian-8.1)
Updates: NetBeans IDE is updated to version NetBeans 8.1 Patch 1
Java: 1.8.0_111; OpenJDK 64-Bit Server VM 25.111-b14
Runtime: OpenJDK Runtime Environment 1.8.0_111-8u111-b14-2ubuntu0.16.04.2-b14
System: Linux version 4.4.0-47-generic running on amd64; UTF-8; en_US (nb)

I installed the plugin from the "Plugin" options in netbeans

Java:
openjdk version "1.8.0_111"
OpenJDK Runtime Environment (build 1.8.0_111-8u111-b14-2ubuntu0.16.04.2-b14)
OpenJDK 64-Bit Server VM (build 25.111-b14, mixed mode)

Scala:
Scala code runner version 2.11.6 -- Copyright 2002-2013, LAMP/EPFL

**Comment by akochnev** [comment]
Yeah, it does look like Ubuntu packaged NetBeans 8.1 patch 1 (8.1.1), whereas the download from netbeans.org is 8.1.0, which is what the current scala plugin is compiled against. Unfortunately, it looks like 8.1.1 was never released as an available versions on the netbeans maven repositories, so we can't really build another version of the plugin against 8.1.1 . 

Your best bet at this point is to uninstall the netbeans versions that you install from Ubuntu , and download a version directly from netbeans.org.

**Comment by akochnev** [comment]
@dcaoyuan - can you please close this issue, as there is unlikely to be a real resolution to it ?

**Comment by scalahub** [comment]
Thanks for the updates. I have resolved the problem by using NetBeans 8.0.2 and it works perfectly.
